src/sass/gi/partials/_gi-preview-banner.scss

```diff
@@ -27,8 +25,11 @@
       margin: 0;
 
       .actions {
-        white-space: nowrap;
-        margin: auto 2em;
+        @media screen and (min-width: $medium-screen) {
+          bottom: 0;
+          position: absolute;
```

Comment:
At a specific width, the banner gets garbled. A little larger or a little smaller looks fine, though. ![image](https://cloud.githubusercontent.com/assets/12970166/24936958/18d18ace-1ee2-11e7-95cc-b6a06a142029.png)